types/puppeteer/index.d.ts

```diff
@@ -106,7 +119,7 @@ export interface Evalable {
   $eval<R>(
     selector: string,
     pageFunction: (element: Element, ...args: any[]) => R | Promise<R>,
```

Comment:
`args` here should have value as well.

Reply:
Or rather, the same type but without `JSHandle` and `Element` instead.

Reply:
Hmm, I thought about that, but `JSHandle`s are not only handles for `Element`s, right? I could pass a `JSHandle` to a Regex, and then my `pageFunction` would be given the regex itself. Is that right?